unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarModifierAreaHUD/Test/AvatarModifierAreaFeedbackShould.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using DCL;
+using DCL.AvatarModifierAreaFeedback;
+using NUnit.Framework;
+using Tests;
+using UnityEngine;
+
+public class AvatarModifierAreaFeedbackShould
```

Comment:
As I suggested above, we should have an interface for the `view` so we can mock it and only test the controller here.

Then I would create a whole test suite for the `view` alone.

Reply:
Done!